static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -282,41 +280,78 @@ class WidgetCard extends Component<Props, State> {
                   windowWidth={windowWidth}
                   onDataFetched={this.setData}
                 />
-              </LazyLoad>
-            )}
-            {this.renderToolbar()}
-          </WidgetCardPanel>
-          <Feature organization={organization} features={['dashboards-mep']}>
-            <DashboardsMEPConsumer>
-              {({isMetricsData}) =>
-                showStoredAlert &&
-                widget.widgetType === WidgetType.DISCOVER &&
-                isMetricsData === false && (
-                  <StoredDataAlert showIcon>
-                    {tct(
-                      "Your selection is only applicable to [storedData: stored event data]. We've automatically adjusted your results.",
-                      {
-                        storedData: <ExternalLink href="https://docs.sentry.io/" />, // TODO(dashboards): Update the docs URL
-                      }
-                    )}
-                  </StoredDataAlert>
-                )
-              }
-            </DashboardsMEPConsumer>
-          </Feature>
-          <Feature organization={organization} features={['dashboards-releases']}>
-            {showIncompleteDataAlert && (
-              <StoredDataAlert showIcon>
-                {tct(
-                  'Releases data is only available from [date]. Data may be incomplete as a result.',
-                  {
-                    date: <DateTime date={METRICS_BACKED_SESSIONS_START_DATE} dateOnly />,
+              ) : (
+                <LazyLoad once resize height={200}>
+                  <WidgetCardChartContainer
+                    api={api}
+                    organization={organization}
+                    selection={selection}
+                    widget={widget}
+                    isMobile={isMobile}
+                    renderErrorMessage={renderErrorMessage}
+                    tableItemLimit={tableItemLimit}
+                    windowWidth={windowWidth}
+                    onDataFetched={this.setData}
+                  />
+                </LazyLoad>
+              )}
+              {this.renderToolbar()}
+            </WidgetCardPanel>
+            <Feature organization={organization} features={['dashboards-mep']}>
+              <DashboardsMEPConsumer>
+                {({isMetricsData}) => {
+                  if (
+                    isMetricsData === false &&
+                    widget.widgetType === WidgetType.DISCOVER
+                  ) {
+                    if (showStoredAlert && isCustomMeasurementWidget(widget)) {
```

Comment:
We can probably move `showStoredAlert` to the if statement on lines 308-311 so we don't have to check it twice here

Reply:
good point!